docker/local/Dockerfile

```diff
@@ -19,14 +19,16 @@ RUN echo "deb http://httpredir.debian.org/debian jessie main" > /etc/apt/sources
  && apt-get install -y --no-install-recommends \
 	bash-completion \
 	curl \
+	dkms \
```

Comment:
I haven't added it. If you look at a few lines below you will see i just resort it :)